docs/software/gaming.md

```diff
@@ -349,4 +350,46 @@ Make sure you have one of DietPi's File Servers installed.
 - Steam is installed to the DietPi user data directory `/mnt/dietpi_userdata/steam`
 - By default, installing games will install them to `/mnt/dietpi_userdata/steam`
 
+## PaperMC - Fast and Optimized Minecraft Server, Extendable with Plugins
+
+Descent 1 & 2. A masterpiece 3D FPS. Brought back to life with the DXX-Rebirth project. Play Descent as true to the original with OpenGL/ES rendering.
+
+- DietPi will install the Demo and Shareware versions of Descent. Please see the FAQ below to transfer the full game.
+- Keyboard + Mouse is highly recommended
+- We have compiled the latest version of DXX-Rebirth (0.58.1) with support for FB and RPi OpenGL.
+
+![DietPi gaming software DXX-Rebirth](../assets/images/dietpi-software-games-papermc.jpg){: style="width:500px"}
+
+=== "Info"
+
+    PaperMC by default runs a single server, available on LAN, but can be port forewarded so other people can connect.
+
+=== "Tweaking"
+
+    Tweak the server settings by modifying the following file or any file that ends in `.yml`:  
+    `/opt/papermc/server.properties`
+
+=== "Restart service"
+
+    You can restart the service by running:  
+    `dietpi-services restart`
+
+### FAQ
+
+#### How do I interface with the server's console?
+
+Use the installed tool McRCON.  `mcrcon -p GLOBAL_PW`
+
+#### How do I find and install plugins?
+
+<http://https://www.spigotmc.org/resources/categories/spigot.4//>
+
+Just move the downloaded JAR file to the `/opt/papermc/plugins` directory.
+
+#### What version of Minecraft does this work on
+
+PaperMC is built to run on Java edition, however with the optional Geyser and Floodgate plugins, it can run on Bedrock edition as well.
+
+Just choose to install them at the beginning
```

Comment:
Add '.'
Just choose to install them at the beginning
->
Just choose to install them at the beginning.